File: python/code forces/KefaAndFirstSteps.py
n = int(input())
nums = list(map(int,input().split()))
counts = [1]
id = 0
while id < len(nums)-1:
    count = 0
    for j in range(id,n):
        try:
            if nums[j] <= nums[j+1]:
                count += 1
            else:
                break
        except:
            break
    id = j+1
    counts.append(count+1)
print(max(counts))

File: C++/codeforces/KefaAndFirstSteps.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int n;cin>>n;
    cin.ignore();
    
    string _;getline(cin,_);
    stringstream ss(_);
    vector<int>nums;
    string num;
    while (ss >> num)
    {        
        nums.push_back(stoi(num));
    }
    vector<int> counts = {1};
    int id = 0;
    while (id < nums.size()-1)
    {
        int count = 0;
        for (id;id<n;id++)
        {
            try
            {
                if (nums.at(id) <= nums.at(id+1)){count++;}
                else{break;}
            }
            catch (...)
            {
                break;
            }
        }
        id++;
        counts.push_back(count+1);
    }
    
    cout << *max_element(counts.begin(),counts.end());

    return 0;
}